lib/models/AuthAccess.js

```diff
@@ -13,16 +13,20 @@ export default class AuthAccess {
 
   constructor(payload) {
     Utils.assertRequiredParameterIsPresent(payload, 'payload');
+    Utils.assertRequiredParameterIsPresent(payload.username, 'username');
     Utils.assertRequiredParameterIsPresent(payload.accessToken, 'accessToken');
-    Utils.assertRequiredParameterIsPresent(payload.api, 'api');
-    Utils.assertRequiredParameterIsPresent(payload.eventSource, 'eventSource');
-    Utils.assertRequiredParameterIsPresent(payload.upload, 'upload');
-    Utils.assertRequiredParameterIsPresent(payload.download, 'download');
+    Utils.assertRequiredParameterIsPresent(payload.apiUrl, 'apiUrl');
+    Utils.assertRequiredParameterIsPresent(payload.eventSourceUrl, 'eventSourceUrl');
+    Utils.assertRequiredParameterIsPresent(payload.uploadUrl, 'uploadUrl');
+    Utils.assertRequiredParameterIsPresent(payload.downloadUrl, 'downloadUrl');
```

Comment:
The exception thrown in `Utils.assertRequiredParameterIsPresent()` already states the failing property name in its message. I assume that's enough information to track down these assertion errors.